libs/application/core/src/lib/ApplicationTemplateHelper.ts

```diff
@@ -49,6 +54,20 @@ export class ApplicationTemplateHelper<
     )
   }
 
+  getApplicationStatus(): ApplicationStatus {
+    const { state } = this.application
+
+    if (this.template.stateMachineConfig.states[state].type === 'final') {
+      if (state === 'rejected') {
```

Comment:
Should this use [`ApplicationStatus.REJECTED`](https://github.com/island-is/island.is/blob/f7554be35dcd15eb138f5eeb2e264b0957fea061/libs/application/core/src/types/Application.ts#L7)?

Perhaps it would even be better to create an enum for `FinalState` that would be used here.